tidy3d/components/simulation.py

```diff
@@ -901,6 +934,12 @@ def set_plot_params(boundary_edge, lim, side, thickness):
         ax.set_xlim([ulim_minus, ulim_plus])
         ax.set_ylim([vlim_minus, vlim_plus])
 
+        # clean up the axis display
+        axis, position = Box.parse_xyz_kwargs(x=x, y=y, z=z)
```

Comment:
worth combining these three lines in a single `Box` method? Seems they're almost always called together.